File: C/PRAK504-2210817310009-MarioFrancaWijaya-.c
#include <stdio.h>

int reverse(int a){
    int reversed = 0;
    while(a!=0){
        reversed = reversed * 10;
        reversed = reversed + a%10;
        a = a/10;
    }
    return reversed;
}
int main(){
    int A, B;
    scanf("%d %d", &A, &B);
    A = reverse(A);
    B = reverse(B);
    int c =  A + B;
    printf("%d", reverse(c));
}

File: Python/PRAK504-2210817310009-Mario Franca Wijaya.py
def reverse(reversed):
    i = 0
    while reversed != 0:
        i = (10 * i) + reversed % 10
        reversed //= 10
    return i

A, B = input().split()
A = reverse(int(A))
B = reverse(int(B))
C = A + B
print(f'{reverse(C)}\n')